diff --git a/dist/node/chunk-D3CUZ4fa.js b/dist/node/chunk-D3CUZ4fa.js
index fcc35e8af6f0178bd475a90d3e51efdb0d80d2c5..31018d54c308c368f7f2d92f1a2fc00578f80e4f 100644
--- a/dist/node/chunk-D3CUZ4fa.js
+++ b/dist/node/chunk-D3CUZ4fa.js
@@ -45110,15 +45110,12 @@ async function createVitePressPlugin(siteConfig, ssr = false, pageToHashMap, cli
           if (isPageChunk(chunk)) {
             const hash = chunk.fileName.match(hashRE)[1];
             pageToHashMap[chunk.name.toLowerCase()] = hash;
-            bundle[name + "-lean"] = {
-              ...chunk,
+            this.emitFile({
+              type: "asset",
+              name: name + "-lean",
               fileName: chunk.fileName.replace(/\.js$/, ".lean.js"),
-              preliminaryFileName: chunk.preliminaryFileName.replace(
-                /\.js$/,
-                ".lean.js"
-              ),
-              code: chunk.code.replace(staticStripRE, `""`)
-            };
+              source: chunk.code.replace(staticStripRE, `""`)
+            });
             chunk.code = chunk.code.replace(staticRestoreRE, "");
           }
         }
